composer.json

```diff
@@ -58,7 +58,7 @@
     },
     "classmap": [
       "Api/",
-      "custom/Extension/Api"
+      "custom/Extension/application/Ext/Api"
```

Comment:
Shouldn't this be
custom/application/Ext/Api
rather than the original file that generates the above file after a repair and rebuild?

Reply:
Changed.